packages/lesswrong/components/common/KarmaDisplay.tsx

```diff
@@ -14,13 +20,27 @@ const KarmaDisplay = ({document, placement="left"}: {
     ? document.afBaseScore
     : null;
   const {LWTooltip} = Components;
+
+  const commentId = !!annualReviewMarketCommentId ? annualReviewMarketCommentId : undefined
```

Comment:
Once https://github.com/ForumMagnum/ForumMagnum/pull/8701/files#r1472219264 is addressed, I don't think this line is necessary at all